src/library/scala/collection/Seq.scala

```diff
@@ -724,6 +727,7 @@ trait SeqOps[+A, +CC[_], +C] extends Any
   def sortBy[B](f: A => B)(implicit ord: Ordering[B]): C = sorted(ord on f)
 
   /** Produces the range of all indices of this sequence.
+    * $willForceEvaluation
```

Comment:
`indices` could be lazy if it was allowed to return a `Seq[Int]` instead of a `Range`